crawler.js

```diff
@@ -592,9 +610,9 @@ export class Crawler {
 
     await this.initPages();
 
-    if (this.params.blockAds) {
-      this.adBlockRules = new AdBlockRules(this.captureBasePrefix, this.params.adBlockMessage, this.logger);
-    }
+    //if (this.params.blockAds) {
```

Comment:
Should we just remove this conditional instead of commenting it out?

Reply:
Yes